### PR TITLE
Issue#60

### PR DESCRIPTION
**Pull request by Leo24**
Amend "counrty" to "county"
Add functionality for saving address info

### CODE REVIEW

File: src/Adapters/FHIRPatientAdapter.php [hunk 1]
```diff
@@ -452,6 +465,22 @@ public function interfaceToModel( PatientInterface $patient )
         $fhirPatient->addTelecom( $email );
 
 
+        $address = new FHIRAddress();
+        $line = new FHIRString();
+        $line->setValue($patient->getStreet());
```
Comment (kchapple) [inline src/Adapters/FHIRPatientAdapter.php:470]: @Leo24 You don't add both lines here if an array is returned by getStreet()

**Review by kchapple — CHANGES_REQUESTED** [review]

File: src/Adapters/FHIRPatientAdapter.php (hunk 1 above)
Comment (Leo24) [reply src/Adapters/FHIRPatientAdapter.php:470]: I do not fully understand you.
Do you mean, I should return only first line from array?
like this:
        $street = $patient->getStreet();
        $line->setValue($street[0]);
        $address->addLine($line);

Comment (kchapple) [reply src/Adapters/FHIRPatientAdapter.php:470]: @Leo24 No, The iPhone UI has two lines, address-line-1 and address-line-2. We are taking those two lines and storing them in the LibreEHR database like this "address-line-1 | address-line-2" with a "|" delimiter. When we get the address out of the LibreEHR database and return it through the FHIR API to the iPhone, we need to reassemble the address into two lines (two entries in the lines array) so there should be two calls to addLine($line) one for address-line-1 and one for address-line-2.

Comment (Leo24) [reply src/Adapters/FHIRPatientAdapter.php:470]: Oh.
Now I understood. Thank you for detailed clarification.
Current getStreet() returns array of address lines.

Do we need to add keys to address lines?
Like this 
"line": [
        [
          "address-line-1" : "Street Value 1",
          "address-line-2" : "Street Value 2"
        ]

patient_adresses

{
  "resourceType": "Patient",
  "extension": [
    {
      "extension": [
        {
          "valueString": "",
          "url": "#groupId"
        },
        {
          "valueString": "active",
          "url": "#status"
        },
        {
          "valueString": 34,
          "url": "#providerId"
        },
        {
          "valueString": 5,
          "url": "#pharmacyId"
        },
        {
          "valueString": "Some value of stripeToken",
          "url": "#stripeToken"
        }
      ],
      "url": "http://libre-emr.loc/fhir/extension/gponline-patient-data"
    }
  ],
  "identifier": [
    {
      "use": "usual",
      "value": 75
    }
  ],
  "name": [
    {
      "use": "usual",
      "family": [
        "Dow"
      ],
      "given": [
        "Judy Simple"
      ]
    }
  ],
  "telecom": [
    {
      "system": "phone",
      "value": "555-555-2003",
      "use": "primary"
    },
    {
      "system": "email",
      "value": "",
      "use": "primary"
    }
  ],
  "gender": "female",
  "birthDate": "1955-01-06",
  "address": [
    {
      "line": [
        [
          "address-line-1",
          "address-line-2"
        ]
      ],
      "city": "Sent-Petersburg",
      "district": "USA"
    }
  ]
}

Comment (kchapple) [reply src/Adapters/FHIRPatientAdapter.php:470]: @Leo24 Your PostMan screenshot is correct.

**Review by kchapple — APPROVED** [review]

**Pull request merged**